runtime/frequency/src/lib.rs

```diff
@@ -202,6 +206,7 @@ impl frame_system::Config for Runtime {
 	/// The identifier used to distinguish between accounts.
 	type AccountId = AccountId;
 	/// Base call filter to use in dispatchable.
+	// enable for cfg feature "frequency" only
```

Comment:
Is this a note to add a feature flag here?

Reply:
Also it's a dev comment // only in case anyone adds new features added to frequency, mainnet should still be filtered until we remove it when we remove sudo